content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class 1,2,3,3a,4,5,6 grey
+class S,T spacewhite
+class changes,changes2 white
+```
+
+Figure 2. Working from a local fork to make your changes.
+
+#### Fork the opentelemetry.io repository
+
+1. Navigate to the
+   [`opentelemetry.io`](https://github.com/open-telemetry/opentelemetry.io/)
+   repository.
+1. Select **Fork**.
+
+#### Create a local clone and set the upstream
+
+1. In a terminal window, clone your fork and install the requirements:
+
+   ```shell
+   git clone git@github.com:<your_github_username>/opentelemetry.io.git
+   cd opentelemetry.io
+   npm install
+   ```
+
+1. Set the `open-telemetry/opentelemetry.io` repository as the `upstream`
+   remote:
+
+   ```shell
+   git remote add upstream https://github.com/open-telemetry/opentelemetry.io.git
+   ```
+
+1. Confirm your `origin` and `upstream` repositories:
+
+   ```shell
+   git remote -v
+   ```
+
+   Output is similar to:
+
+   ```none
+   origin	git@github.com:<your_github_username>/opentelemetry.io.git (fetch)
+   origin	git@github.com:<your_github_username>/opentelemetry.io.git (push)
+   upstream	https://github.com/open-telemetry/opentelemetry.io.git (fetch)
+   upstream	https://github.com/open-telemetry/opentelemetry.io.git (push)
+   ```
+
+1. Fetch commits from your fork's `origin/main` and
+   `open-telemetry/opentelemetry.io`'s `upstream/main`:
+
+   ```shell
+   git fetch origin
+   git fetch upstream
+   ```
+
+   This makes sure your local repository is up to date before you start making
+   changes.
+
+#### Create a branch
+
+1. Create a new branch. This example assumes the base branch is `upstream/main`:
+
+   ```shell
+   git checkout -b <my_new_branch> upstream/main
+   ```
+
+1. Make your changes using a code or text editor.
+
+At any time, use the `git status` command to see what files you've changed.
+
+#### Commit your changes
+
+When you are ready to submit a pull request, commit your changes.
+
+1. In your local repository, check which files you need to commit:
+
+   ```shell
+   git status
+   ```
+
+   Output is similar to:
+
+   ```none
+   On branch <my_new_branch>
+   Your branch is up to date with 'origin/<my_new_branch>'.
+
+   Changes not staged for commit:
+   (use "git add <file>..." to update what will be committed)
+   (use "git checkout -- <file>..." to discard changes in working directory)
+
+   modified:   content/en/docs/contribute/new-content/contributing-content.md
+
+   no changes added to commit (use "git add" and/or "git commit -a")
+   ```
+
+1. Add the files listed under **Changes not staged for commit** to the commit:
+
+   ```shell
+   git add <your_file_name>
+   ```
+
+   Repeat this for each file.
+
+1. After adding all the files, create a commit:
+
+   ```shell
+   git commit -m "Your commit message"
+   ```
+
+1. Push your local branch and its new commit to your remote fork:
+
+   ```shell
+   git push origin <my_new_branch>
+   ```
```

Comment:
GitHub these days sends back a message that you can use to create a PR from here, do we want to call this out?